src/core/mount.c

```diff
@@ -1312,15 +1311,14 @@ static void mount_sigchld_event(Unit *u, pid_t pid, int code, int status) {
                 if (f == MOUNT_SUCCESS && m->from_proc_self_mountinfo) {
 
                         /* Still a mount point? If so, let's try again. Most likely there were multiple mount points
-                         * stacked on top of each other. Note that due to the io event priority logic we can be sure
-                         * the new mountinfo is loaded before we process the SIGCHLD for the mount command. */
+                         * stacked on top of each other. */
 
                         if (m->n_retry_umount < RETRY_UMOUNT_MAX) {
                                 log_unit_debug(u, "Mount still present, trying again.");
                                 m->n_retry_umount++;
                                 mount_enter_unmounting(m);
                         } else {
-                                log_unit_debug(u, "Mount still present after %u attempts to unmount, giving up.", m->n_retry_umount);
+                                log_unit_error(u, "Mount still present after %u attempts to unmount, giving up.", m->n_retry_umount);
```

Comment:
i am fine with raising this, but we generally use LOG_WARN for stuff we don't fail fatally on, rather than LOG_ERR. Hence, please let's use LOG_WARN HERE too, as we just accept our fate here and proceed.

Reply:
It's a bit subtle, as I think the _job_ will fail...

This can be the root cause of a cascading failure, e.g. during shutdown.  Does that argue for making it an error?

Reply:
dunno, if in doubt always go for the lower level I'd say...